src/core/track-scroll-depth.ts

```diff
@@ -10,6 +10,17 @@ import { EventTimer } from './event-timer';
 const initTrackScrollDepth = () => {
 	const pageHeight = document.body.offsetHeight;
 	const intViewportHeight = window.innerHeight;
+
+	// this if statement is here to handle a bug in Firefox in Android where the innerHeight
+	// of a new tab can be 0, so we end up dividing by 0 and looping through infinity
+	if (intViewportHeight === 0) {
```

Comment:
Just chiming to ensure we capture other “invalid values”. I think you’d probably want to discard any value that’s below 1, such as 0.75 or 0.000001, for example, which would yield to `pageHeightVH` to be greater than `pageHeight`. Not sure if this could actually happen, so feel free to disregard!

Reply:
Doesn't `window.innerHeight` only return integer values?

Reply:
It should do… unless there's a bug in the implementation… like returning 0 😜

Reply:
0 is of type integer though, so although it causes issues here it isn't a breach in type. Given this, I feel like getting a value between 0 and 1 is pretty unlikely.